app/src/sample/java/com/mycompany/myapp/ui/main/SampleViewModel.kt

```diff
@@ -0,0 +1,74 @@
+package com.mycompany.myapp.ui.main
+
+import android.app.Application
+import android.arch.lifecycle.AndroidViewModel
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import com.mycompany.myapp.BuildConfig
+import com.mycompany.myapp.app.MainApplication
+import com.mycompany.myapp.data.api.github.GitHubService
+import com.mycompany.myapp.data.api.github.GitHubService.LoadCommitsRequest
+import com.mycompany.myapp.data.api.github.GitHubService.LoadCommitsResponse
+import rx.schedulers.Schedulers
+import rx.subscriptions.CompositeSubscription
+import javax.inject.Inject
+
+class SampleViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
Would like to only extend ViewModel and not AndroidViewModel, but couldn't figure out the dependency injection.  Need help 😄